test_weatherman.py

```diff
@@ -0,0 +1,102 @@
+import weatherman
+import os
+from zipfile import ZipFile
+
+
+with ZipFile("/Users/hassandanial/W/weatherfiles.zip", 'r') as zip_file:
+    # Extract all the contents of zip file in current directory
+    zip_file.extractall()
+
+dir_list = [f for f in os.listdir(
+    "./weatherfiles") if f.endswith('.txt')]
+
+Years = []
+Yearwithmonth = []
```

Comment:
We use snake case as a convention for python language. Your global variables can be updated to. 
```suggestion
YEARS = []
YEARS_WITH_MONTH = []
```
Please make sure to use snake cases for all variables except classes